arch/arm64/boot/dts/qcom/msm8953-xiaomi-vince.dts

```diff
@@ -137,6 +137,24 @@
 	status = "okay";
 };
 
+&i2c_3 {
+	status = "okay";
+
+	touchscreen@1 {
```

Comment:
downstream nvt node is not on @1
https://github.com/xiaomi-msm8953-devs/android_kernel_xiaomi_msm8953/blob/1ea7e46cc1d6635bb716a3effffb949bcacb11ad/arch/arm64/boot/dts/qcom/xiaomi/vince/vince-qrd.dtsi#L201C3-L201C13

`touchscreen@62` should be the right reg

Reply:
Can't imagine touchscreen driver should work though if `reg` is wrong? Unless the driver is weirdly hardcoding the i2c address

Reply:
> Can't imagine touchscreen driver should work though if `reg` is wrong? Unless the driver is weirdly hardcoding the i2c address

I think it does something like this it is not working with 0x62, but the real i2c address is 0x62.